commitlint.config.js

```diff
@@ -15,9 +15,10 @@ module.exports = {
         'scope-min-length': [2, 'always', 3],
         // Body
         'body-min-length': [2, 'always', 30],
-        'body-max-line-length': [2, 'always', 100],
+        'body-max-line-length': [2, 'always', 130],
         'body-leading-blank': [2, 'always'],
     },
+    ignores: [ (message) => message.includes('[skip-lint]') ],
```

Comment:
how does it work?

Reply:
this one enable us to skip the commit lint, maybe it's need to be renamed.

the idea is that some commit message, like ones having backtraces, enforcing 100 max body line length is too much
this give a way to skip it, in certain cases.

yet again, we might just remove the line length limit  for body

Reply:
I disagree to increase the message body limit.
I used line continuations in such cases.

The problem is that lots of people will start abusing this limit absence in common cases.

All lengths must fit half of the 1920p screen, IMHO.

Reply:
I'm tired of manually edit backtrace to fit, and anyhow I'm putting it with triple backtick.

which make it readable just fine in github 

half screen ? where did you came up with that ?

other repo we don't have those limitation, and we are just find with it., people are no abusing them in that sense.

my requirement is, I want to be able to paste backtrace right into the commit.
and in some case it's important to see it all (like the example in this PR)

Reply:
> half screen ? where did you came up with that ?

Split screen approach.